playbooks/roles/registry-server-setup/tasks/main.yml

```diff
@@ -18,6 +18,7 @@
   tags:
     - install
 
+# TODO(itxaka): Check if this is still valid on caasp4 as its using crio
```

Comment:
What about creating Jira story instead? IMHO it is not lost so easily as random TODOs...

Reply:
that is mainly because I expected to check that out myself and change the file accordingly. Especially because failures here do not make the deployment fail, Im not sure if this was needed or no. Hence the reminder, for myself :)